src/AzureAuth/Commands/Ado/CommandToken.cs

```diff
@@ -98,6 +98,13 @@ public int OnExecute(ILogger<CommandToken> logger, IEnv env, ITelemetryService t
                 return 0;
             }
 
+            // If command line options for mode are not specified, then use the environment variables.
+            this.AuthModes ??= AuthModeHelper.ReadAuthModeFromEnvOrSetDefault(env, eventData, logger);
+            if (!this.AuthModes.Any())
+            {
+                return 1;

```

Comment:
Should we add a logger call here, similar to what you had before?

ie:` this.logger.LogError($"Invalid value specified for environment variable {EnvVars.AuthMode}. Allowed values are: {AuthModeAllowedValues}");`